Toolkit/ArcGISToolkit/AR/ArcGISARView.swift

```diff
@@ -386,13 +400,22 @@ extension ArcGISARView: SCNSceneRendererDelegate {
         if let camera = arSCNView.session.currentFrame?.camera {
             let intrinsics = camera.intrinsics
             let imageResolution = camera.imageResolution
+            
+            // Get the device orientation, but don't allow non-landscape/portrait values.
+            var deviceOrientation = UIDevice.current.orientation
```

Comment:
It looks like this could be simplified a little bit by making `deviceOrientation` immutable, getting rid of the `else` clause, and using `lastGoodDeviceOrientation` on line `418`.

Reply:
Perfect, thanks.